docs/components/attention/index.md

```diff
@@ -12,7 +12,13 @@
 
 <theme-switcher />
 
-<attention-example></attention-example>
+<attention-example />
+
+## Usage
+
+<component-design-guidelines name="Warp - Components / Callout" link="https://www.figma.com/file/nkiRpuVu6XRfvY96BA80H8/Components-overview?type=design&node-id=259-13731&mode=design" />
```

Comment:
I don't know how a discussion on splitting Attention ended in [this Slack thread](https://sch-chat.slack.com/archives/C050N3NQ204/p1695020570901139), but at this moment Attention component is documented on 3 different pages in Figma:
Callout: https://www.figma.com/file/nkiRpuVu6XRfvY96BA80H8/Components-overview?type=design&node-id=259-13731&mode=design
Tooltip: https://www.figma.com/file/nkiRpuVu6XRfvY96BA80H8/Components-overview?type=design&node-id=377-23911&mode=design
Popover: https://www.figma.com/file/nkiRpuVu6XRfvY96BA80H8/Components-overview?type=design&node-id=374-22825&mode=design

So maybe we should add all 3 links here for now?

Reply:
Ah! Of course! 👍  ...too bad I made it a component that only accepts one label and one link. 🙈 😅  I have to figure something out here...

Reply:
I don't think it makes sense to make the ComponentDesignGuidelines take more links just to cater for one case. It should be fine to apply the header and links inline here, especially as we may end up aligning on the Attention documentation and have 1 page for each of its types also here in the tech-docs.

Reply:
I've added the above mentioned Figma links to the Attention component now. 👍